### DISCUSSION TITLE
how can i see the source code of package 'Microsoft.Extensions.Hosting.Systemd'

### DISCUSSION THREAD

**Discussion by sometiny** [dotnet/runtime#44584] [Q&A] [answered]
i want to view the code and write a  new library that contain Systemd and WindowsService. thank u

**Comment by gfoidl** [comment, marked as answer]
See https://github.com/dotnet/extensions/tree/master/src/Hosting and its subfolders.

**Reply by eerhardt** [reply]
In general, a great tip is to click on the "Source repository" link in nuget.org:

https://www.nuget.org/packages/Microsoft.Extensions.Hosting.Systemd/

![image](https://user-images.githubusercontent.com/8291187/99000622-3175db80-24ff-11eb-98c6-dbd832d1f386.png)

**Reply by sometiny** [reply]
get it , thank you for your help~
